runtime/caml/misc.h

```diff
@@ -166,9 +166,11 @@ CAMLdeprecated_typedef(addr, char *);
 /* Prefetching */
 
 #ifdef CAML_INTERNALS
-#if (__has_builtin(__builtin_prefetch) || defined(__GNUC__)) && \
+#if (__has_builtin(__builtin_prefetch) || defined(__GNUC__) || \
+     defined(__clang__)) && \
```

Comment:
the `__clang__` macro is not needed, clang defines `__GNUC__` and recent version recognize the `__has_builtin` macro.

Reply:
Thanks, I'll simplify that.